classList.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Element.classList utility functions
+ * @module delite/classlist
+ * */
+define([], function () {
+
+	var classList = /** @lends module:delite/classList */ {
+		/**
+		 * Toggle one or multiple classes helper method.
+		 *
+		 * @param {Element} node
+		 * @param {String} value
+		 * @param {Boolean} force
+		 */
+		toggleClass: function (node, value, force) {
+			if (value && value.trim() !== "") {
+				if (force === true) {
+					classList.addClass(node, value);
+				}
+				else if (force === false) {
+					classList.removeClass(node, value);
+				}
+				else {
+					var method;
+					this._getValues(value).forEach(function (v) {
+						method = node.classList.contains(v) ? "removeClass" : "addClass";
+						classList[method](node, v);
+					}, this);
```

Comment:
I don't think we have anywhere that actually wants to toggle the value (based on the current value), do we?  

If we don't, we could get rid of this whole block.

If we do, then the block could be simplified to just use `node.classList.toggle(value)`, since toggle() with a single argument works even on IE11.

Reply:
We do have places where `toggleClass` is called without passing a force argument. 
The reason I'm not using the `classList.toggle` here is because this way we can handle multiple classes like `toggleClass('foo bar')`